### PR TITLE
Deal with missing or bad data

### LINKED ISSUE #517: categories broken on the first 2 posts
## Summary

Categories broken on the first 2 posts. 

## Screenshot

Screenshot 2019-04-08 at 10 35 46

Can you please estimate this issue? thank you

### PR DESCRIPTION
**Pull request by pmahnke**
## Done

- make headers work with categories that do not exist
- make headers work with null categories defaulting to 'Ubuntu'
- make footers add 'Article' if they do not have a type

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8023/](http://0.0.0.0:8023/)
- See that people and culture works ok
- See that footers default to text

## Issue / Card

Fixes #517

## Screenshots

**Header**
image

**Footer**
image

### CODE REVIEW

File: static/sass/_pattern_card.scss [hunk 1]
```diff
@@ -82,6 +82,10 @@
 
   .p-card__header {
     @extend %post-card-header;
+    border-bottom: 0;
+    border-top: 3px solid #666666;
+    margin-bottom: 0;
+    padding: 1rem 1rem .5rem;
```
Comment (deadlight) [inline static/sass/_pattern_card.scss]: Consider using Vanilla spacing variables:
https://docs.vanillaframework.io/en/settings/spacing-settings

**Review by deadlight — CHANGES_REQUESTED** [review]
Try using spacing variables

File: static/sass/_pattern_card.scss (hunk 1 above)
Comment (pmahnke) [reply static/sass/_pattern_card.scss]: good catch... fixed.

**Review by deadlight — APPROVED** [review]
good stuff!

**Pull request merged**